test/e2e/provisioning/testdata/machinedeployment-aws-spot-instances.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: "cluster.k8s.io/v1alpha1"
+kind: MachineDeployment
+metadata:
+  name: << MACHINE_NAME >>
+  namespace: kube-system
+spec:
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+  selector:
+    matchLabels:
+      name: << MACHINE_NAME >>
+  template:
+    metadata:
+      labels:
+        name: << MACHINE_NAME >>
+    spec:
+      providerSpec:
+        value:
+          sshPublicKeys:
+            - "<< YOUR_PUBLIC_KEY >>"
+          cloudProvider: "aws"
+          cloudProviderSpec:
+            accessKeyId: << AWS_ACCESS_KEY_ID >>
+            secretAccessKey: << AWS_SECRET_ACCESS_KEY >>
+            region: "eu-central-1"
+            availabilityZone: "eu-central-1a"
+            vpcId: "vpc-819f62e9"
+            instanceType: "t2.medium"
+            instanceProfile: "kubernetes-v1"
+            diskSize: 50
+            diskType: "gp2"
+            ebsVolumeEncrypted: false
+            ami: "<< AMI >>"
+            isSpotInstance: true
+            spotInstanceConfig:
+              maxPrice: "0.02"
```

Comment:
What is this value actually? Is this percentage, price, or something else?

Reply:
This is actually the price as the name suggests :) so it is 2 cents per hour